src/components/NavDropdown/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Divider } from 'antd';
+import { NavLink } from 'react-router-dom';
+
+import './style.less';
+import NavListItem from '../NavListItem';
+
+const NavDropdown = props => {
+  return (
+    <nav className="nav-dropdown">
+      <section>
+        <NavLink activeClassName="selected" className="nav-item" to="/dashboard">
+          <NavListItem link="/dashboard" text="Dashboard" />
+        </NavLink>
+        <NavLink activeClassName="selected" className="nav-item" to="/leaderboard">
+          <NavListItem text="Leaderboard" />
+        </NavLink>
+        <NavLink activeClassName="selected" className="nav-item" to="/profile">
+          <NavListItem text="Profile" />
+        </NavLink>
+      </section>
+
+      <Divider style={{ margin: '0.5rem 0' }} />
```

Comment:
Not sure if this applies to styles, but AirBnB React styling says no padding in curly braces https://i.imgur.com/cZV1ZBS.png

Reply:
https://github.com/airbnb/javascript/tree/master/react#alignment